### ISSUE TITLE
Custom Attribute Accessors not working

### ISSUE THREAD

**Issue by ellisonpatterson** [DirectoryTree/LdapRecord#83]
**Describe the bug**
I want to set some custom attributes on the Model but building the accessor like in eloquent doesn't seem to work.

At this line:
https://github.com/DirectoryTree/LdapRecord/blob/e40c63e586763e3341f84a48ba7d14996785ec71/src/Models/Concerns/HasAttributes.php#L99
It performs the `getAttribute()` method call which will always return null since it's not checking for accessors, the second method in that if operation is doing it.

I might be dumb but it seems due to this, accessors are not accessible (lol).

Let me know what you think!

**To Reproduce**
Here's my model that I set in the auth.php config as the primary user handler.
```
<?php

namespace App\Models;

use LdapRecord\Models\ActiveDirectory\User as Model;

class User extends Model
{
    public function getEmailAttribute($value)
    {
        return $this->userprincipalname;
    }

    public function isPasswordMatch($password)
    {
        if ($this->getConnection()->auth()->attempt($this->getFirstAttribute('distinguishedname'), $password)) {
            return true;
        }

        return false;
    }
}
```
**Environment (please complete the following information):**
 - LDAP Server Type: AD
 - PHP Version: 7.3

**Comment by stevebauman** [comment]
Hi @ellisonpatterson!

Thanks for the report! This is a bug. Right now, the attribute must exist on the model for the accessor to be called (via the `hasAttribute()` call).

Fixing now :smile:

**Comment by stevebauman** [comment]
Fixed in [v1.2.3](https://github.com/DirectoryTree/LdapRecord/releases/tag/v1.2.3).

Thank you!

**Comment by ellisonpatterson** [comment]
Thank you for fixing it so quickly!
